### ISSUE TITLE
Make lottery registration checkboxes more clear.

### ISSUE THREAD

**Issue by alexcole** [learning-unlimited/ESP-Website#531]
This is spinning some of discussion from Issue #438 into a new issue.

Basically, we are not clearly telling students the entire lottery flow for MIT Splash registration. We give them checkboxes on the reg page, but they don't totally reflect what students need to do.

A more useful set of checkboxes could have short phrases like:
1. Go to the lottery and check off the classes you find interesting (and be sure to read the descriptions!)
2. Wait until foo to get results
3. Modify your schedule below
4. Buy meal tickets (Optional, but must happen before payment)

and so on so that students can consistently see a concise description of the entire flow in the same place.
